js/activity.js

```diff
@@ -1308,11 +1308,8 @@ define(MYDEFINES, function (compatibility) {
                 .setPaste(paste);
 
             languageBox = new LanguageBox();
-            languageBox
-                .setCanvas(canvas)
-                .setStage(stage)
-                .setMessage(textMsg)
-                .setRefreshCanvas(refreshCanvas);
+            languageBox.setMessage(textMsg);
```

Comment:
Use languageBox as the arg here: https://github.com/sugarlabs/musicblocks/blob/0702865ded2fddbb55f585c4beab4abbc30db82f/js/activity.js#L1685
instead of doLanguageBox